testpmd/hooks/files/collect-sriov-operator-data.sh.j2

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash 
```

Comment:
I guess this needs to be renamed to be used as a script and not as a template, right?